### PR TITLE
Merging changes

### PR DESCRIPTION
**Pull request by mershad-manesh**
New additions:
* Allow user to control allowPrivilegeEscalation, allowedCapabilities, defaultAddCapabilities
* Disable the built in Ingress if the className is not set
* A couple of small tweaks and fixes to init scripts

### CODE REVIEW

**Review by deejgregor — COMMENTED** [review]
Here's an initial review--I'll get back to it later after I have some more time, but I wanted to give you these few pieces of feedback sooner.

Also: it looks like the github linter found a few issues, too.

File: horizon/scripts/onms-core-init.sh [hunk 1]
```diff
@@ -517,8 +517,12 @@ else
 fi
 
 echo "Updating admin password"
-if [[ -e "/opt/opennms/bin/password.jar" ]];then 
- java -jar /opt/opennms/bin/password.jar "${CONFIG_DIR}/users.xml" "admin" "${OPENNMS_ADMIN_PASS}"
+if [[ -e "/opt/opennms/bin/password" ]];then 
+ echo "RUNAS=$(whoami)" > /opt/opennms/etc/opennms.conf
+ bash /opt/opennms/bin/runjava -s > /dev/null 2>&1
```
[the remark this answers is not in the document]
Comment (deejgregor) [reply horizon/scripts/onms-core-init.sh:522]: Wait... I wrote `runjava`, and `-q` is something I would have likely added. LOLLL...... I did! :)

LOLZ... there is a `-q` option to `runjava`:
```
  -q  Quiet mode.  Be fairly quiet.
```

[the remark this answers is not in the document]
Comment (mershad-manesh) [reply horizon/scripts/onms-core-init.sh:522]: In this case, we need to use `bash` before the runjava command as the behaviour of the default shell in UBI is slightly different than Ubuntu based images.
The long term solution would be to update runjava script to do a better job in handling different shells

**Review by deejgregor — APPROVED** [review]
LGTM

**Pull request merged**